crates/zk-por-core/benches/batch_circuit.rs

```diff
@@ -1,23 +1,17 @@
 #![feature(test)]
 
 use zk_por_core::{
-    account::gen_accounts_with_random_data,
-    merkle_sum_prover::{
-        circuits::merkle_sum_circuit::build_merkle_sum_tree_circuit, prover::MerkleSumTreeProver,
-    },
+    account::gen_accounts_with_random_data, merkle_sum_prover::prover::MerkleSumTreeProver,
 };
 
 extern crate test;
 use test::Bencher;
 
 fn bench(b: &mut Bencher, batch_size: usize) {
-    let num_assets = 4;
-    let (circuit_data, account_targets) = build_merkle_sum_tree_circuit(batch_size, num_assets);
-    let accounts = gen_accounts_with_random_data(batch_size, num_assets).0;
-    b.iter(|| {
-        let prover = MerkleSumTreeProver { accounts: accounts.clone() };
-        _ = prover.prove_with_circuit(&circuit_data, account_targets.clone());
-    });
+    let num_assets = 50;
+    let accounts = gen_accounts_with_random_data(batch_size, num_assets);
+    let prover = MerkleSumTreeProver { accounts };
```

Comment:
the idea of prove_with_circuit is to avoid build multiple times. copy is lighter compared to build circuit again. can have a comparison between copy and build

Reply:
Yeah will add a version for this